app/main.py

```diff
@@ -1 +1,28 @@
-# write your code here
+from os import mkdir, remove, path, rename
+
+
+def move_file(command: str) -> None:
+    try:
+        mv, file, new_location = command.split()
+        if mv != "mv":
+            raise ValueError
+    except ValueError:
+        raise ValueError(
+            "Incorrect command. Example: mv file1.txt file2.txt "
+            "or mv file1.txt new_folder/file1.txt"
+        )
+
+    if "/" not in new_location and file != new_location:
+        rename(file, new_location)
```

Comment:
why do u need this part of code, if 26-28 lines will do the same?

Reply:
If i delete this part then the program will occur an error, cause if we can only file, without directories, then during splitting in 21 line we'll get an error. [Demonstration](https://prnt.sc/OUQYXXlTtvUG)